anki/ga-autotrack-ids
# Initial Review

@LeeSaferite Please read through the README and do an initial code review and try to run the tests. This is still WIP but may be to do an initial release. Some things that still need to be done:


- Update dependencies to latest versions
- Finish test coverage with unit & e2e tests
- Make sure all original files from [autotrack](https://github.com/googleanalytics/autotrack) which contain apache license in the header and were modified include also a modification notice in the header.
- Add thanks to [autotrack](https://github.com/googleanalytics/autotrack) authors.
- Finish documenting plugins (only one documented so far)
